infrastructure/kubernetes/workflows-default/workflows-default-dask-rolebinding.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: workflows-default-dask-rolebinding
+subjects:
+  - kind: ServiceAccount
+    name: workflows-default
+roleRef:
+  kind: Role
+  name: daskkubernetes
+  apiGroup: rbac.authorization.k8s.io
```

Comment:
So is this the file that updates the workers' permissions?

Reply:
@dgergel Yeah, so we have a `ServiceAccount` (assigned to the Argo Workflow), we have a `Role` (which defines some permissions), and then the `RoleBinding` is the thing that links them all together -- and in the darkness binds them.